test/fixImageSize.js

```diff
@@ -1,23 +1,14 @@
-const sharp = require('sharp');
-const sizeOf = require('image-size');
-const fs = require('fs');
- 
-function getFileSizeMB(fileStats){
-    return (fileStats.size) / (1024*1024);
-}
+import {imageSizeChecker} from './library.js';
+import {getAllFiles} from './library.js';
+import {fixImage} from './library.js';
+
 
-function fixImage(imageFileName){
-    sharp(imageFileName)
-        .resize({width: 512, height: 512})
-        .toFile(imageFileName+ "1").then(()=>{
-        fs.rename(imageFileName+ "1",imageFileName, (err) => {
-            if (err) throw err;
-             let imageSize = sizeOf(input);
-             let width = imageSize.width;
-             let height = imageSize.height;
-             let imageStats = fs.statSync(input);
-             console.log(imageFileName + "has size " + getFileSizeMB(imageStats).toFixed(2)+"MB");
-             console.log(imageFileName + "has dimensions "+width +"px by " +height+"px");
-            });
- });
+const finalFilesArray =  getAllFiles("./img/team");
+for (const file of finalFilesArray){
+    const imageStatus = imageSizeChecker(file);
+    if (imageStatus.error){
+        if(imageStatus.fixable){
+            fixImage(file);
```

Comment:
What happens if the fixing fails? We don't catch any errors or print it.

Do we keep track of how many files we fix?